live-site/components/welcome-section/team-race/Team.styles.ts

```diff
@@ -0,0 +1,46 @@
+import styled from 'styled-components';
+import { P } from '../../../../shared-ui/style/typography';
+
+const StyledTeamContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  padding: 1em 2em;
+`;
+
+const StyledTeamName = styled(P)`
+  text-transform: uppercase;
+  color: #bed9aa;
```

Comment:
We don't want to hard code colors when we're using them so change this to use colors.WHATEVERCOLOR since all colors on our website should be stored in the colors object